src/index.js

```diff
@@ -143,11 +143,20 @@ app.ports.logError.subscribe((msg, err) => {
   if (env === 'development') {
     console.error(msg, err)
   } else {
+    let error = 'Generic Elm Error port msg'
+    let details = ''
+
+    if (Object.prototype.toString.call(msg) === '[object Array]') {
+      [error, details] = msg
+    }
+    Object.prototype.toString.call(msg)
```

Comment:
Looks like this string is redundant. Or maybe this expression should be assigned to a variable?

Reply:
thanks for catching that!